pxc/jenkins/qa_pxc_cross-pipeline.groovy

```diff
@@ -0,0 +1,180 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/kaushikpuneet07/jenkins-pipelines.git'
+]) _
+
+pipeline {
+   agent {
+        label 'docker'
+    }
+  parameters {
+    string(name: 'LOWER_PXC_VERSION', defaultValue: '5.7.38-31.59', description: 'PXC lower version tarball to download for testing')
+    string(name: 'UPPER_PXC_VERSION', defaultValue: '8.0.27-18.1', description: 'PXC Upper version tarball to download for testing')
```

Comment:
Why not update the PXc version to 8.0.28

Reply:
Updated , Fixed